README.md

```diff
@@ -1,30 +1,85 @@
 # NUpbr
+
 This repository holds code to generate a configurable, Physically Based Rendered (PBR) football field.
 
 ![Field Example](./docs/outputs/goals_example.gif)
 
 # Requirements
-+ [Blender](https://www.blender.org/download/) (2.79)
+
+-   [Blender](https://www.blender.org/download/) (2.79)
 
 # Usage
-## Generating field UV map
-A UV map for the field can be generated by running [`generate_uv.py`](./field_uv/generate_uv.py). This will use the configuration listed in [`scene_config.py`](./pbr/config/scene_config.py).
 
-## Building the scene
-To generate the scene with default field UV map, the `pbr` script should be run through the Blender Python API. That is, `blender --python pbr.py` while in the pbr directory. This will create a scene, rendering a ball, goals and a field depending on the HDR metadata.
+## Set Up
+
+-   Clone this repo
+-   Change into the cloned `NUpbr` directory, then into `pbr`
+-   Install dependencies by running `ensure_dependencies.py` _using the Python binary installed with Blender_. You may have to run it twice: first to install Pip, and then to install the dependencies.
+-   Download the latest [resources.zip](https://github.com/NUbots/NUpbr/releases) file from Releases and copy the `resources` directory from it into the `NUpbr` root directory. More resources are available on the NUbots NAS.
+
+## Building a Scene
+
+To generate a scene with default field UV map, do the following:
+
+-   Change into the `pbr` directory
+-   Run `pbr.py` using Blender's Python API: `blender --python pbr.py`
+-   To run the script without the Blender UI, use: `blender -b --python pbr.py`
 
-The ball UV map and HRDI environment image are randomly selected from the respective configured directory, within [`scene_config.py`](./pbr/config/scene_config.py).
+This will create a scene, rendering a ball, goals and a field depending on the HDR metadata. The output files will be placed in `output/run_#` where `#` is the run number.
```

Comment:
Just so people dont think they have to specify the run number themselves or something.

```suggestion
This will create a scene, rendering a ball, goals and a field depending on the HDR metadata. The output files will be placed in `output/run_#` where `#` is the auto-generated run number.
```